spec/mongoid/association/referenced/has_many_models.rb

```diff
@@ -36,6 +44,9 @@ class HmmSchool
 class HmmStudent
   include Mongoid::Document
 
+  belongs_to :current_academy, class_name: 'HmmAcademy', inverse_of: :students, optional: true
+  belongs_to :previous_academy, class_name: 'HmmAcademy', inverse_of: nil, optional: true
```

Comment:
I think these should also go into a new model, unless there is something else from HmmStudent that your test would need.

Reply:
I can create a dedicated model for this yes.